evalml/tests/model_understanding_tests/prediction_explanations_tests/test_algorithms.py

```diff
@@ -93,7 +89,7 @@ def calculate_shap_for_test(training_data, y, pipeline, n_points_to_explain):
     return _compute_shap_values(pipeline, pd.DataFrame(points_to_explain), training_data)
 
 
-interpretable_estimators = [e for e in _all_estimators_used_in_search() if e.model_family not in {ModelFamily.XGBOOST, ModelFamily.BASELINE}]
+interpretable_estimators = [e for e in _all_estimators_used_in_search() if e.model_family not in {ModelFamily.BASELINE}]
```

Comment:
not necessary and very nit, but why not change to `e.model_family != ModelFamily.BASELINE` just to make it cleaner since it's only 1 object we're checking for

Reply:
Good catch!